### DISCUSSION TITLE
Possible to rollback into failed deploys?

### DISCUSSION THREAD

**Discussion by JVFP** [deployphp/deployer#3661] [Help needed]
Hi,

It seems the rollback code checks for a BAD_RELEASE file inside the previous release to see whether or not we can rollback to that release. Maybe I'm missing something but I only see BAD_RELEASE files being created in the current release during a rollback, never during a failed deploy.

If during a failed deploy it doesn't automatically create a BAD_RELEASE file inside it, wouldn't that mean on a rollback I could end up rolling back into a failed deploy?

**Comment by antonmedv** [comment]
I need to check this. Don’t remember the behaviour right now.
